engine/runtime/src/main/scala/org/enso/interpreter/runtime/IrToTruffle.scala

```diff
@@ -1020,10 +1032,11 @@ class IrToTruffle(
     */
   sealed private class ExpressionProcessor(
     val scope: LocalScope,
-    val scopeName: String
+    val scopeName: String,
+    private val initialName: String
```

Comment:
Can we add a `@param` doc explaining what this argument is?

Reply:
Technically our [style guide](https://github.com/enso-org/enso/blob/f1a19010586eabe1d643a193f45ef3639c6bf070/docs/style-guide/scala.md#documentation-comments) requires to document all parameters if any one of them is non-obvious.

Reply:
Sure: d332b74